### DISCUSSION TITLE
TurboHook: Doc typo.

### DISCUSSION THREAD

**Discussion by yuri-kiss** [TurboWarp/extensions#601] [Q&A] [answered]
![image](https://github.com/TurboWarp/extensions/assets/135030944/a6e07d2e-32ab-48aa-aad6-8e4c55771dfe)
unless it was intended, which is why its not in issues.

**Comment by GarboMuffin** [comment, marked as answer]
that's a typo

**Reply by yuri-kiss** [reply]
lol so it needs to be un-typo;ed?
